lib/policies_finder_publisher.rb

```diff
@@ -67,10 +67,31 @@ def details
         document_type: "policy"
       },
       show_summaries: false,
-      facets: [],
+      facets: facets,
     }
   end
 
+  def facets
+    [
+      {
+        key: "public_timestamp",
+        short_name: "Updated",
+        name: "Published",
```

Comment:
Yeah, both need thinking about, i'd be OK with merging this and updating
them later too

On 6 May 2015 at 17:29, Tommy Palmer <notifications@github.com> wrote:

> In lib/policies_finder_publisher.rb
> <https://github.com/alphagov/policy-publisher/pull/60#discussion_r29775686>
> :
>
> >      }
> >    end
> >
> > +  def facets
> > +    [
> > +      {
> > +        key: "public_timestamp",
> > +        short_name: "Updated",
> > +        name: "Published",
>
> Now that I think about it, maybe that should be changed too. Not sure if
> there will be confusion with the difference. Not a blocker for merging
> anyways.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/policy-publisher/pull/60/files#r29775686>.
>



-- 
*David Singleton*

Cabinet Office, Government Digital Service
Developer, GOV.UK
david.singleton@digital.cabinet-office.gov.uk